frame/staking/src/lib.rs

```diff
@@ -1096,6 +1115,7 @@ decl_module! {
 				let era = Self::current_era().unwrap_or(0) + T::BondingDuration::get();
 				ledger.unlocking.push(UnlockChunk { value, era });
 				Self::update_ledger(&controller, &ledger);
+				Self::deposit_event(RawEvent::Unbonded(stash.clone(), value));
```

Comment:
```suggestion
				Self::deposit_event(RawEvent::Unbonded(ledger.stash.clone(), value));
```
I do see a reason to have a one time used variable `stash` here.